tools/unitctl/unitctl/src/cmd/instances.rs

```diff
@@ -12,12 +12,23 @@ pub(crate) async fn cmd(args: InstanceArgs) -> Result<(), UnitctlError> {
                 ref application,
                 ref image
             } => {
+                println!("Pulling and starting a container from {}", image);
+                println!("Will mount {} to /var/run for socket access", socket);
```

Comment:
What is being mounted there by who and how?

Or is this inside the container?

Reply:
This is inside the container. As is expected if you read the help text for the command that this codepath is in.